DelayedExecution/UpdateHandle.cs

```diff
@@ -42,7 +42,7 @@ public class UpdateHandle : AbstractAnvilBase
         // Seems likely that an application wouldn't need more than 100 more delayed calls before it can
         // create a new UpdateHandle to work with.
         // If this is an issue there's no risk to having the threshold trigger sooner.
-        private readonly IDProvider m_IDProvider = new IDProvider();
+        private readonly UintIDProvider m_IDProvider = new UintIDProvider();
```

Comment:
I think this comment is outdated, from when this IDProvider instance had an explicit 100-ID's-left warning

Reply:
Good point. I've removed the comment completely. The defaults in the providers are enough to go on.